raiwidgets/raiwidgets/error_analysis_dashboard.py

```diff
@@ -73,15 +73,50 @@ class ErrorAnalysisDashboard(Dashboard):
     :param num_leaves: The number of leaves of the surrogate tree
         trained on errors.
     :type num_leaves: int
+    :param sample_dataset: Used only if explanation is not specified for
```

Comment:
Maybe this docstring can start with a description of the parameter? Then go on to say when it's used?

Something like: "Dataset with fewer samples than the main dataset. Used to improve performance only when an Explanation object is not provided."?

Reply:
done, improved the description!